tools/nl/embeddings/data/curated_input/sheets_svs.csv

```diff
@@ -1520,7 +1520,7 @@ dc/topic/Emissions_PM2.5_EmissionsbySource,Emissions of PM2.5 By Source,Emission
 dc/topic/Energy_Generation_FossilFuels,Energy Generation From Fossil Fuels,Energy Generation From Fossil Fuels,,energy produced from fossil fuels;
 dc/topic/Fire,Fires,Fires,,fire; fire event; wild fires;
 dc/topic/GreenhouseGasEmissions,Greenhouse Gas Emission,Greenhouse Gas Emissions,,GHG emissions; greenhouse gasses emissions; sources of greenhouse gas emissions; sources of GHG emissions;
-dc/topic/Gender,Gender,Gender,,
+dc/topic/Gender,Gender,Gender,,gender of population;
```

Comment:
what does ";" mean in this csv?

Reply:
";" is the separator for within each csv cell

Reply:
in other words, a semi colon is used as a separator between alternative sentences/descriptions etc.